app/src/modules/settings/components/navigation.vue

```diff
@@ -63,12 +65,15 @@ const externalItems = computed(() => {
 		{
 			icon: 'bug_report',
 			name: t('report_bug'),
-			href: 'https://github.com/directus/directus/issues/new?template=bug_report.yml',
+			href:
+				settings.value?.project_bug_url ?? 'https://github.com/directus/directus/issues/new?template=bug_report.yml',
```

Comment:
Was already like this, but I'd welcome it if we could move these URLs to `app/src/constants.ts` 😃

Reply:
Cleanup is always good! Done via https://github.com/directus/directus/pull/21174/commits/a9d2cb3ae2d8fc928509a1d602a9de43f702b864